tests/app/services/test_response_formatters.py

```diff
@@ -32,30 +32,30 @@ def test_should_build_search_response_from_es_response(services_mapping):
     assert_equal(res["meta"]["query"]["q"], "keywords")
     assert_equal(res["meta"]["total"], 628)
     assert_equal(res["meta"]["took"], 69)
-    assert_equal(len(res["services"]), 10)
-
-    assert_equal(res["services"][0]["id"], "5390159512076288")
-    assert_equal(res["services"][0]["lot"], "SaaS")
-    assert_equal(res["services"][0]["frameworkName"], "G-Cloud 6")
-    assert_equal(res["services"][0]["supplierName"], "Supplier Name")
-    assert_equal(res["services"][0]["serviceName"], "Email Verification")
-    assert_equal(res["services"][0]["serviceTypes"], [
+    assert_equal(len(res["documents"]), 10)
+
+    assert_equal(res["documents"][0]["id"], "5390159512076288")
+    assert_equal(res["documents"][0]["lot"], "SaaS")
+    assert_equal(res["documents"][0]["frameworkName"], "G-Cloud 6")
+    assert_equal(res["documents"][0]["supplierName"], "Supplier Name")
+    assert_equal(res["documents"][0]["serviceName"], "Email Verification")
+    assert_equal(res["documents"][0]["serviceTypes"], [
```

Comment:
If you fancied it you could replace these nose assertions (your call)

Reply:
Yep, done.